promptsource/templates/piqa/templates.yaml

```diff
@@ -1,54 +1,88 @@
 dataset: piqa
 templates:
-  1a1d13ee-2ef5-4c80-89dc-b9e003d4ef45: !Template
+  16e97a16-c958-4956-bfba-279f88dafd5b: !Template
     answer_choices: null
     answer_choices_key: null
-    id: 1a1d13ee-2ef5-4c80-89dc-b9e003d4ef45
-    jinja: 'Given a goal and a correct solution, generate a similar but wrong solution.
+    id: 16e97a16-c958-4956-bfba-279f88dafd5b
+    jinja: 'Goal: {{goal}}
 
-      Goal: {{goal}}
 
-      Correct solution: {{[sol1, sol2][label]}}
+      Which is the correct ending?
 
-      Wrong solution:
+      - {{sol1}}
 
-      |||
+      - {{sol2}}
 
-      {{[sol1, sol2][1 - label]}}
 
-      '
+      Answer:
+
+      |||
+
+      {{[sol1, sol2][label]}}'
     metadata: !TemplateMetadata
-      _do_eval: false
-      _do_train: true
-      choices_in_prompt: null
-      metrics: []
-      original_task: null
-    name: Generate a similar but wrong solution
+      choices_in_prompt: true
+      metrics:
+      - Accuracy
+      original_task: true
+    name: what_is_the_correct_ending
     reference: ''
-  45773b41-4d13-4ade-9102-afe1543f5776: !Template
+  3f336295-c1f7-410a-8fc6-d2ed79487aa4: !Template
     answer_choices: null
     answer_choices_key: null
-    id: 45773b41-4d13-4ade-9102-afe1543f5776
-    jinja: 'Given a goal and 2 solutions, choose the most appropriate solution.
+    id: 3f336295-c1f7-410a-8fc6-d2ed79487aa4
+    jinja: '{{"Solution 1"}}: {{sol1}}
+
+      {{"Solution 2"}}: {{sol2}}
+
 
       Goal: {{goal}}
 
-      {{"Solution 1"}}: {{sol2}}
 
-      {{"Solution 2"}}: {{sol1}}
+      Given the goal, what is the correct solution?
+
 
       Answer:
 
       |||
 
       {{[sol1, sol2][label]}}'
     metadata: !TemplateMetadata
-      _do_eval: false
-      _do_train: false
       choices_in_prompt: true
-      metrics: []
-      original_task: null
-    name: 'choose the most appropriate solution: reorder solution'
+      metrics:
+      - Accuracy
+      original_task: true
+    name: pick_correct_choice_with_choice_given_before_goal
```

Comment:
disambiguate as too whether the model has to output `Solution 1` or the text in Solution 1